crates/iota-analytics-indexer/README.md

```diff
@@ -0,0 +1,125 @@
+IOTA Analytics Indexer
+=======================
+
+The IOTA Analytics Indexer is a service that exports data from the main IOTA network to a remote big object store (S3/GCS/Azure) for further analytical processing. It does not perform any analysis on its own.
+
+**Key Features**
+----------------
+
+* Exports data from the IOTA network to a remote big object store
+* Provides BigQuery and Snowflake schemas for the exported data
+
+> Note: BigQuery and Snowflake are cloud-based data warehousing solutions.
+> After getting data there one can analyse it in the cloud using SQL queries.
+> 
+> BigQuery is part of Google Cloud Platform: [https://cloud.google.com/bigquery]
+> 
+> Snowflake isn't part of any large cloud provider: [https://snowflake.com]
+
+
+**Relation to iota-indexer**
+----------------------------
+
+### iota-indexer
+
+Currently iota-indexer is computing and storing analytical metrics about:
+ - network statistics (amount of transactions, transactions per second)
+ - (active) addresses (transactions senders/recipients)
+ - move calls
+
+ Those metrics are computed by a separate analytical worker instance of the indexer, but it uses the main DB as the main indexer instance.
+
+It seems that some of the values stored in main indexer tables by iota-indexer's `fullnode_sync_worker` are only stored there for analytical purposes (move calls, tx recipients) and could potentially be not processed/stored if not for analytical reasons. 
+
+### iota-analytics-indexer
+
+The `iota-analytics-indexer` is not computing any analytical metrics directly.
+It is only exporting data for further processing via external tools (BigQuery/SnowFlake).
+Functionality from `iota-indexer` that is not required to serve user JSON RPC/GraphQL requests could potentially be moved away from `iota-indexer` and served by some other tool based on data exported by the `iota-analytics-indexer`.
+The sync logic in `iota-indexer` could be simplified a bit then to store only data that is needed to serve user requests.
+
+
+
+**Schemas**
+-----------
+
+The crate provides:
+ - [BigQuery Schemas](src/store/bq/schemas/)
+ - [SnowFlake Schemas](src/store/snowflake/schemas/)
+ - [Rust struct representations](src/tables.rs)
+
+for the data that it is exporting.
+
+The tables covered by the schemas:
+ - CHECKPOINT
+ - EVENT
+ - MOVE_CALL
+ - OBJECT
+ - MOVE_PACKAGE
+ - TRANSACTION_OBJECT - input and output objects for given transactions
+ - TRANSACTION
+
+
+> Note: The following rust structs currently do not have DB schemas prepared:
```

Comment:
Nit: For further emphasis (see also [GH alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts))
```suggestion
> [!NOTE]
> The following rust structs currently do not have DB schemas prepared:
```